src/hyperlight_guest/third_party/musl/arch/x86_64/bits/float.h

```diff
@@ -4,6 +4,22 @@
 #define FLT_EVAL_METHOD 0
 #endif
 
+#ifdef _MSC_VER
```

Comment:
Do we need this one now since we are not use MSVC any longer?

Reply:
I believe we need it, because without it clang-cl gives a bunch of warnings about floating point magitudes